Backend/src/main/java/com/onbank/api/controller/TransferController.java

```diff
@@ -37,10 +40,13 @@
     @ResponseStatus(HttpStatus.OK)
     public void createTransfer(@Valid @RequestBody CreateTransferDto transferDto) {
         Transfer tmpTransfer = TransferTransformer.convertToEntity(transferDto);
+
+        Authentication auth = SecurityContextHolder.getContext().getAuthentication();
```

Comment:
Ta logika nie powinna znajdować się w kontrolerze, kontroler tylko ma za zadanie przygotować dane i przesłać je do serwisu, jeżeli potrzebujesz kontekstu uzytkownika to powinieneś go odebrać już w serwisie